src/System.Memory/src/System/MemoryDebugView.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+namespace System
+{
+    internal sealed class MemoryDebugView<T>
```

Comment:
Do we really need both debug views?  Couldn't the same one be used for both types, just with two different constructors?

Reply:
I wasn't sure if the same DebugView class can be applied to different types. You are right, we do not need both.